strawberry/arguments.py

```diff
@@ -55,7 +53,7 @@ def get_arguments_from_resolver(
 
     if len(arguments_missing_annotations) > 0:
         raise MissingArgumentsAnnotationsError(
-            field_name, arguments_missing_annotations
+            resolver.__name__, arguments_missing_annotations
```

Comment:
Discussion: This is no longer the camelcase-d resolver name. I figure this is actually preferable, as this error will occur at startup. The camelcase name would only be helpful if this was a runtime error

This also means this function now only needs one argument